worker/models/http_response.go

```diff
@@ -0,0 +1,32 @@
+package models
```

Comment:
donot bulk all the models in single file. Segregate them based on domain i.e. users, emails etc